extras/rya.pcj.fluo/pcj.fluo.api/src/main/java/org/apache/rya/indexing/pcj/fluo/api/CreatePcj.java

```diff
@@ -139,6 +141,7 @@ public CreatePcj(final int spInsertBatchSize) {
      * @throws QueryEvaluationException Historic results could not be added to the initialized PCJ because of
      *   a problem with the Rya connection.
      */
+    @VisibleForTesting
```

Comment:
artifact left over from when I added the withRyaIntegration() that returned the generated table name